apps/ima/translations/src/en/pages.json

```diff
@@ -54,26 +54,29 @@
     "header": "Declaration"
   },
   "immigration-adviser-declaration": {
+    "adviser": "Immigration adviser declaration",
```

Comment:
I don't think this is needed here, the title is already showing as 'Declaration'

Reply:
Removed